datastore/core/query.py

```diff
@@ -235,7 +235,7 @@ def __repr__(self):
         return "Order('%s%s')" % (self.op, self.field)
 
     def __eq__(self, other):
-        return self.field == other.field and self.op == other.op
+        return self.field == other.type and self.op == other.op
 
     def __ne__(self, other):
         return not self.__eq__(other)
```

Comment:
Oh, and this is totally unnecessary as that's actually the default implementation `__ne__` :slightly_smiling_face:

Reply:
Cool